resources/views/speakers/create.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>Add Speakers</h1>
+            @auth
```

Comment:
Whats doing here this `@auth`?

Reply:
so far @auth is doing that a user who is not logged in will not enter it

Reply:
Just like the other blade

Reply:
maybe middleware will be better ? :>

Reply:
better but @Majkulnn  still working for auth and not have middleware with working